src/go/rpk/pkg/system/syslog/syslog.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// +build linux
+
+package syslog
+
+import (
+	"fmt"
+	"syscall"
```

Comment:
This should use the x/sys/unix package, as syscall is deprecated.

Reply:
Fixed.